lib/extensions/firebase_extensions.dart

```diff
@@ -3,19 +3,13 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 import '../logic/models/seal.dart';
 
 extension SealDocs on List<QueryDocumentSnapshot> {
-  List<Seal> toSeals() {
-    final sealsList = <Seal>[];
-    for (final QueryDocumentSnapshot snapshot in this) {
-      sealsList.add(
-        Seal(
+  List<Seal> toSeals() => map<Seal>(
```

Comment:
I changed this to a functional approach, just because I was playing around here caus this was the code where the old version (with null checks) gave a warning locally, and the new code (without null checks) errored in the build in CI.  It's a bit neater imo, happy to change it back if you prefer though!

Reply:
I like it, it's nice and clean 👍